packages/config/src/layer2s/canvasconnect.ts

```diff
@@ -0,0 +1,123 @@
+import { ProjectId, UnixTime } from '@l2beat/shared'
+
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import {
+  getCommittee,
+  getProxyGovernance,
+  getSHARPVerifierContracts,
+  getSHARPVerifierGovernors,
+} from '../discovery/starkware'
+import { delayDescriptionFromString } from '../utils/delayDescription'
+import { formatSeconds } from '../utils/formatSeconds'
+import {
+  CONTRACTS,
+  DATA_AVAILABILITY,
+  EXITS,
+  FORCE_TRANSACTIONS,
+  makeBridgeCompatible,
+  NEW_CRYPTOGRAPHY,
+  NUGGETS,
+  OPERATOR,
+  RISK_VIEW,
+  STATE_CORRECTNESS,
+} from './common'
+import { Layer2 } from './types'
+
+const discovery = new ProjectDiscovery('canvasconnect')
+const delaySeconds = discovery.getContractUpgradeabilityParam(
+  'StarkExchange',
+  'upgradeDelay',
+)
+const delay = formatSeconds(delaySeconds)
+const verifierAddress = discovery.getAddressFromValue(
+  'GpsFactRegistryAdapter',
+  'gpsContract',
+)
+
+export const canvasconnect: Layer2 = {
+  type: 'layer2',
+  id: ProjectId('canvasconnect'),
+  display: {
+    name: 'Canvas Connect',
+    slug: 'canvasconnect',
+    warning:
+      'Canvas Connect is initially open only to whitelisted institutional clients.',
+    description:
+      'Canvas Connect is a Layer 2 ZK network built on the StarkEx technology that aims to offer privacy-focused solutions for the finance and digital economy. It aims to enable the transfer, trade, and investment of tokenized real-world assets, stablecoins, CBDCs, and digital assets with assurances of privacy, low cost, and high speed.',
+    purpose: 'Privacy, Finance',
+    links: {
+      websites: ['https://canvas.co/'],
+      apps: [],
+      documentation: ['https://docs.starkware.co/starkex/index.html'],
+      explorers: [],
+      repositories: ['https://github.com/starkware-libs/starkex-contracts'],
+      socialMedia: [
+        'https://twitter.com/canvas_defi',
+        'https://canvasdefi.medium.com/',
+        'https://linkedin.com/company/canvasblockchaingroup',
+        'https://canvas.co/content',
+      ],
+    },
+  },
+  config: {
+    escrows: [
+      discovery.getEscrowDetails({
+        identifier: 'StarkExchange',
+        sinceTimestamp: new UnixTime(1675209600),
+        tokens: ['ETH', 'USDC'],
+      }),
+    ],
+  },
+  riskView: makeBridgeCompatible({
```

Comment:
Are you sure about the risks? maybe some references to source code?

Reply:
I'm doing a bulk for all starkex, let me include it there